### DISCUSSION TITLE
Select query, join table which is not directly referenced via Foreign Key

### DISCUSSION THREAD

**Discussion by tomfox747** [prisma/prisma#14905] [Q&A] [answered]
With Prisma, I'm trying to join an indirectly referenced table onto a query, see SQL example below which represents the basic use case.
Lets say in the example, that I want to access the companyCar table from the companyUser table. I can think of 2 ways to do this, via joining using the company table, or by directly comparing the companyId of the 2 tables

**option 1, follows foreign key references**
SELECT * FROM CompanyUser companyuser
JOIN Company company on company.Id = companyuser.companyId
JOIN CompanyCar companycar on companycar.companyId = company.Id

**option 2, does not directly follow foreign key references**
SELECT * FROM companyUser companyuser
JOIN CompanyCar companycar on companycar.companyId = companyuser.companyId

How do I solve this within the prisma framework using includes or some other mechanism?
Thanks :) 

I should probably add, that the foreign keys aren't necessary, so **to simplify the question, can I join 2 tables without the use of foreign keys, using Prisma?**

**Comment by austincrim** [comment, marked as answer]
Hey @tomfox747 👋,

This isn't possible with Prisma yet, though we have an [outstanding feature request](https://github.com/prisma/prisma/issues/13517) for it. Feel free to add your specific use case as that will help our engineers prioritize features in the future.

In the meantime, you can fall back to [$queryRaw](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#queryraw) to accomplish this.

**Reply by tomfox747** [reply]
Thank you for answering my question.

I should be able to utilize queryRaw for the time being.
Other than this, I'm loving the framework overall so far, keep up the good work!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
